snapcraft/plugins/rust.py

```diff
@@ -102,6 +103,40 @@ def build(self):
             cmd.append(' '.join(self.options.rust_features))
         self.run(cmd, env=self._build_env())
 
+    def _write_config(self):
+        if not self.project.is_cross_compiling:
+            return
+
+        # Cf. http://doc.crates.io/config.html
+        with suppress(FileExistsError):
+            os.mkdir(os.path.join(self.builddir, '.cargo'))
```

Comment:
I prefer os.makedirs(..., exist_ok=True). Was it @kyrofa who prefered suppress? Any reason for that?